<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icon/res.qrc">
    <normaloff>:/db/db_connect.png</normaloff>:/db/db_connect.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="grp_sourceDB">
     <property name="title">
      <string>Database Connection Parameters</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_50">
      <item row="2" column="0">
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="0">
       <spacer name="verticalSpacer_8">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>5</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="0">
       <layout class="QGridLayout" name="gridLayout_48">
        <item row="0" column="1">
         <layout class="QGridLayout" name="gridLayout_45">
          <item row="3" column="1">
           <spacer name="verticalSpacer_16">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="1" column="0">
           <layout class="QGridLayout" name="gridLayout_15">
            <item row="0" column="0">
             <widget class="QLabel" name="lbl_dbName_2">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Username   :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QLineEdit" name="edt_user">
              <property name="text">
               <string/>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_12">
            <item row="0" column="0">
             <widget class="QLabel" name="lbl_dbName">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>DB Instance :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QComboBox" name="cb_dBName">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="currentIndex">
               <number>0</number>
              </property>
              <item>
               <property name="text">
                <string>demo.db</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SWGD</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SWGS</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SWGX</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>SWGP</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="lbl_dbName_6">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>The user's password</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <layout class="QGridLayout" name="gridLayout_18">
            <item row="0" column="0">
             <widget class="QLabel" name="lbl_dbName_3">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>90</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Password   :</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <layout class="QGridLayout" name="gridLayout_11">
              <item row="0" column="0">
               <widget class="QLineEdit" name="edt_password">
                <property name="text">
                 <string/>
                </property>
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QCheckBox" name="chk_dbSave">
                <property name="text">
                 <string>Save</string>
                </property>
                <property name="checked">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item row="0" column="1">
           <widget class="QLabel" name="lbl_dbName_4">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Instance name of the server</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLabel" name="lbl_dbName_5">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>110</width>
              <height>0</height>
             </size>
            </property>
            <property name="text">
             <string>Name of the user to connect with</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <widget class="QTreeWidget" name="tw_dbList">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>9</pointsize>
           </font>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="indentation">
           <number>20</number>
          </property>
          <property name="allColumnsShowFocus">
           <bool>true</bool>
          </property>
          <attribute name="headerMinimumSectionSize">
           <number>24</number>
          </attribute>
          <attribute name="headerDefaultSectionSize">
           <number>100</number>
          </attribute>
          <attribute name="headerStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>User Name</string>
           </property>
           <property name="textAlignment">
            <set>AlignCenter</set>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </item>
      <item row="3" column="0">
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="btn_dbConnection">
          <property name="text">
           <string>Connection</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="btn_cancel">
          <property name="text">
           <string>Cancel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>cb_dBName</tabstop>
  <tabstop>edt_user</tabstop>
  <tabstop>edt_password</tabstop>
  <tabstop>chk_dbSave</tabstop>
  <tabstop>btn_dbConnection</tabstop>
  <tabstop>btn_cancel</tabstop>
  <tabstop>tw_dbList</tabstop>
 </tabstops>
 <resources>
  <include location="icon/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>edt_user</sender>
   <signal>returnPressed()</signal>
   <receiver>btn_dbConnection</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>420</x>
     <y>80</y>
    </hint>
    <hint type="destinationlabel">
     <x>502</x>
     <y>422</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>edt_password</sender>
   <signal>returnPressed()</signal>
   <receiver>btn_dbConnection</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>382</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>504</x>
     <y>419</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
